src/invidious/routes/before_all.cr

```diff
@@ -100,6 +88,24 @@ module Invidious::Routes::BeforeAll
       end
     end
 
+    unregistered_path_whitelist = {"/", "/login", "/licenses", "/privacy"}
```

Comment:
I think `/videoplayback` needs to be added, otherwise I was not able to watch videos (DASH).